### DISCUSSION TITLE
Galeria de fotos  vivareal

### DISCUSSION THREAD

**Discussion by jeisontc87** [frontendbr/forum#2305] [Perguntas e Respostas]
Olá pessoal tudo bem?
Alguém sabe onde encontro um codigo fonte de galeria de imóveis parecido com o do site de imoveis vivareal? 
Quando vc clica pra ver detalhes do imovel em monitores grandes a galeria de fotos é no formato grid, e  quando vc no celular é no formato slider.
Estou criando um site de imóveis e gostaria muito de usar esse tipo de galeria

**Comment by felquis** [comment]
Manda o link para a gente ver a galeria por favor.

**Reply by jeisontc87** [reply]
Opa! O link é esse, repare como no desktop a galeria abre como grid e no celular ela abre como slide

https://www.vivareal.com.br/imovel/apartamento-3-quartos-novo-mundo-bairros-curitiba-com-garagem-67m2-venda-RS340000-id-2522878143/

**Reply by felquis** [reply]
Entendi

**Comment by felquis** [comment]
Talvez algo com o [swiperjs](https://swiperjs.com/) te ajude um pouco, ele tem opções como:

- slides-per-view https://swiperjs.com/demos#slides-per-view
- slide grids https://swiperjs.com/demos#slides-grid que é bem parecido com o que você quer.

Os exemplos acima tem links para ver o código usado então da para copiar e colar para experimentar.

A parte de mobile e desktop, talvez vc vai ter que usar um method matchMedia https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia para ver se o tamanho da tela está para configuração 1 ou 2 do swiperjs.

Usei em 2 projetos recentemente e não tive reclamações,
enfim espero que te ajude! o/

**Reply by jeisontc87** [reply]
Vou dar uma olhada, muito obrigado pela ajuda !
